src/mixins/form-radio-check.js

```diff
@@ -82,7 +82,8 @@ export default {
     },
     is_Required() {
       // Required only works when a name is provided for the input(s)
-      return Boolean(this.get_Name && this.bvGroup.required)
+      // Child can be required while parent isn't, but is always disabled if group is
+      return Boolean(this.get_Name && (this.bvGroup.required || this.required))
```

Comment:
A radio or check button in a group cannont be "required" when the group is not "required".  It is either the whole group is required or not.

If the check or radio is not part of a group, then it can have the required attribute on it's own (although it doesn't make sense for individual radios, as radio's must be grouped - tied to same variable - to be functional).

```js
return Boolean(this.get_Name && (this.bvGroup.is_Group() ? this.bvGroup.required : this.required))
```